scipy/stats/_distribution_infrastructure.py

```diff
@@ -3533,7 +3533,17 @@ def make_distribution(dist):
                 ``endpoints`` tuple above, and should define a subinterval of the
                 domain given by ``endpoints``.
 
-            A ``tuple`` value ``(a, b)`` is equivalent to ``{endpoints: (a, b)}``.
+            A ``tuple`` value ``(a, b)`` associated to a key in the ``parameters``
+            ``dict`` is equivalent to ``{endpoints: (a, b)}``.
+
+            Custom distributions with multiple parameterizations can be defined by
+            having the ``parameters`` attribute be a ``tuple`` of ``dict``s with
+            the structure described above. In this case, ``dist``'s class must also
+            define a method ``process_parameters` to map between the different
+            parameterizations. It must take all parameters from all parameterizations
+            as optional keyword arguments and return a ``dict`` mapping parameters to
+            values, filling in values from other parameterizations using values from
+            the supplied parameterization.
```

Comment:
```suggestion
            the supplied parameterization. See example.
```
While this is about as clear as it can be, it's worth mentioning to the confused reader that there is an example coming.

Reply:
100% agreed. I felt a little confused writing this even.